AndroidCompat/getAndroid.sh

```diff
@@ -20,15 +20,15 @@ fi
 
 
 # foolproof against running from AndroidCompat dir instead of running from project root
-if [ "$(basename $(pwd))" = "AndroidCompat" ]; then
+if [ "$(basename "$(pwd)")" = "AndroidCompat" ]; then
   cd ..
 fi
 
 
 echo "Getting required Android.jar..."
 rm -rf "tmp"
 mkdir -p "tmp"
-pushd "tmp"
+pushd "tmp" || exit
```

Comment:
what does this line do?

Reply:
If pushd fails(folder doesnt exist or similar) it will exit the script instead of continuing

Reply:
why would it fail?!

Reply:
Maybe no permissions? It was a lint in IntelliJ